packages/mdc-textfield/index.js

```diff
@@ -75,7 +78,37 @@ export class MDCTextfield extends MDCComponent {
           label.classList.remove(className);
         }
       },
-    }, this.getInputAdapterMethods_(), this.getHelptextAdapterMethods_()));
+    }, this.getInputAdapterMethods_(), this.getHelptextAdapterMethods_(), this.getBottomLineAdapterMethods_()));
```

Comment:
I wonder if we should break up MDC Textfield the same way we did Tabs? We could have a separate foundation for input, help text, bottom line?